x/evidence/exported/evidence.go

```diff
@@ -34,10 +34,10 @@ type ValidatorEvidence interface {
 	GetTotalPower() int64
 }
 
-// MsgSubmitEvidence defines the specific interface a concrete message must
+// MsgSubmitEvidenceI defines the specific interface a concrete message must
 // implement in order to process submitted evidence. The concrete MsgSubmitEvidence
 // must be defined at the application-level.
-type MsgSubmitEvidence interface {
+type MsgSubmitEvidenceI interface {
```

Comment:
Why do we still need an interface here?

Reply:
And why the renaming?

Reply:
> Why do we still need an interface here?

basically:
- MsgSubmitEvidence is the wire type
- MsgSubmitEvidenceI is the domain type, which the keeper manipulates

not strictly necessary, but i also don't want to introduce too many refactors here

> And why the renaming?

it's confusing to have an interface and a struct having the same name.